templates/blog/product-cards/_desktop_ubuntu-desktop-for-organisations.html

```diff
@@ -0,0 +1,18 @@
+<div class="p-card js-product-card u-hide">
+    <img class="p-card__thumbnail"
+         src="https://assets.ubuntu.com/v1/9e27d14a-Ubuntu%20logo@2x.png"
+         alt="ubuntu logo" />
+    <hr class="u-sv1" />
```

Comment:
With the latest vanilla version we either have `p-rule` or `p-rule--muted` class with `<hr>` tags. Could you please amend this for all hr tags in the PR